src/Taylor1.jl

```diff
@@ -981,6 +981,133 @@ function atanHomogCoef{T<:Number}(kcoef::Int, ac::Array{T,1}, rc::Array{T,1}, co
     coefhomog
 end
 
+### HYPERBOLIC TRIGONOMETRIC FUNCTIONS ###
```

Comment:
"TRIGONOMETRIC" should be erased; the functions are simply called hyperbolic functions.